CONTRIBUTION.md

```diff
@@ -0,0 +1,235 @@
+with external systems or APIs.
+
+### Running Tests
+
+Unit testing is essential for software development. It ensures individual code components work 
+correctly, catches bugs early, and improves code reliability. This section explains how to run unit 
+tests for each SDK module.
+
+#### Execute unit tests for forgerock-core
+
+   ```bash
+    ./gradlew :forgerock-core:testDebugUnitTestCoverage --stacktrace --no-daemon
+   ```
+
+#### Execute unit tests for forgerock-auth
+
+   ```bash
+    ./gradlew :forgerock-auth:testDebugUnitTestCoverage --stacktrace --no-daemon
+   ```
+
+#### Execute unit tests for forgerock-authenticator
+
+   ```bash
+    ./gradlew :forgerock-authenticator:testDebugUnitTestCoverage --stacktrace --no-daemon
+   ```
+
+#### Execute unit tests for ping-protect
+
+   ```bash
+    ./gradlew :ping-protect:testDebugUnitTestCoverage --stacktrace --no-daemon
+   ```
+
+### Build API Reference Documentation
+
+Comprehensive and accurate API reference documentation is essential for developers working with the
+Ping SDK for Android. It serves as the definitive guide to the SDK's classes, methods, and
+functions, enabling developers to quickly understand how to utilize its capabilities effectively.
+This section outlines the process for generating API reference documentation directly from the
+source code.
+
+You can build the API reference documentation, which uses Dokka to generate either Javadoc or HTML 
+output, using the following commands:
+
+HTML
+: `./gradlew clean dokkaHtmlMultiModule`
+: View the output at [`build/api-reference/html/`](build/api-reference/html/index.html).
+
+JavaDoc
+: `./gradlew clean dokkaJavadocCollector`
+: View the output at [`build/api-reference/javadoc/`](build/api-reference/javadoc/index.html).
+
+> **TIP**: Use the following command to build both HTML and JavaDoc:
+>
+>`./gradlew clean dokkaHtmlMultiModule dokkaJavadocCollector`
+
+## 2. Standards of Practice
+
+We ask that all contributors to this project adhere to our engineering Standard for team culture, practices and code of conduct. We expect everyone to be respectful, inclusive, and collaborative. Any violations will be handled according to the project's guidelines.
+
+For more details on our Standards of Practice, please refer to the [SDK Standards of Practice](https://github.com/ForgeRock/sdk-standards-of-practice) documentation.
+
+## 3. Creating a Pull Request (PR)
+
+Once you've implemented your changes, the next step is to submit them for review by creating a 
+Pull Request (PR). A PR is a formal request to merge your changes from your forked repository into 
+the main project. This section outlines the process for creating a well-structured PR that 
+facilitates efficient review and integration of your contributions. By following these guidelines, 
+you ensure your code is presented clearly and is ready for consideration.
+
+### Step-by-Step Guide
```

Comment:
Might not need this header. We've said you'll be following the guidelines above, so I think you can go straight into the numbered headings.